pages/api/v1/fetch-follows/followersAndFollowingInfo.ts

```
@@ -0,0 +1,106 @@
+}
+
+async function handler(req: NextApiRequest, res: NextApiResponse<FollowersResponse>) {
+	storeApiKeyUsage(req);
+
+	const network = String(req.headers['x-network']);
+	if (!network || !isValidNetwork(network)) {
+		return res.status(400).json({ message: 'Missing or invalid network name in request headers', followers: [], following: [] });
+	}
+
+	const { userId } = req.body;
+	if (isNaN(Number(userId)) || userId === null || userId === undefined) return res.status(400).json({ message: 'Missing userId in request body', followers: [], following: [] });
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)